core/play/src/main/scala/play/api/http/HttpErrorHandler.scala

```diff
@@ -499,10 +502,11 @@ class JsonHttpErrorHandler(environment: Environment, sourceMapper: Option[Source
  */
 object DefaultHttpErrorHandler
     extends DefaultHttpErrorHandler(HttpErrorConfig(showDevErrors = true, playEditor = None), None, None) {
-  private lazy val setEditor: Unit = {
-    val conf = Configuration.load(Environment.simple())
-    conf.getOptional[String]("play.editor").foreach(setPlayEditor)
-  }
+  private lazy val setEditor: Unit =
+    Try(Configuration.load(Environment.simple())) match {
+      case Success(conf) => conf.getOptional[String]("play.editor").foreach(setPlayEditor)
+      case Failure(_)    => // Very likely invalid config, not able to set the editor
+    }
```

Comment:
Would it make sense to log the failure with some helpful message in the case of `Failure(_)`? How about:

```suggestion
  private val logger = Logger(getClass)
  private lazy val setEditor: Unit =
    Try(Configuration.load(Environment.simple())) match {
      case Success(conf) => conf.getOptional[String]("play.editor").foreach(setPlayEditor)
      case Failure(t) => // Very likely invalid config, not able to set the editor
        logger.error("Failed to load the configuration setup. This uusally means there's a syntax error in your conf files.", t)
    }
```

PS: I haven't tested the code in the suggestion

Reply:
Thanks @ignasi35 - I added a error message but changed the wording a little bit so it talks about `play.editor`, I think otherwise it's too generic.